### ISSUE TITLE
Maybe a 24 hours format time and datetime

### ISSUE THREAD

**Issue by SomosAMambo** [BKWLD/decoy#72]
Hi Guys,

In our effort to localize Decoy, we sometimes find some caveats achieving this goal. Time format HH:MM (AM||PM) isn't a international standard. It would be lovely if we can use 24 hours format in addition to this.

**Comment by weotch** [comment]
Does #71 fix this or are there other instances of hard coded date formatting you've noticed that need changing?

**Comment by SomosAMambo** [comment]
No, it doesn't. This is in the time and datetime field types. As soon as we have the time to tackle this ourselves, we'll send a new PR.
